src/contact-details/age-helpers.tsx

```diff
@@ -0,0 +1,71 @@
+// Based on http://www.gregoryschmidt.ca/writing/patient-age-display-ehr-conventions,
+// which is different from npm packages such as https://www.npmjs.com/package/timeago
+export function age(dateString: string): string {
```

Comment:
This looks familiar. I guess you copied it from widgets (or somewhere else)?

Maybe we could move it into esm-core (we have a utils package in there where common date formatting utilities should be placed). Thanks!

:beers:

Reply:
@FlorianRappl let's merge this PR and I get this copied over to esm-core and make a new PR on esm-core and here?

Reply:
Or just use it already.

`age` is exported from `@openmrs/esm-framework`.

Reply:
Alright